components/BundleModal.tsx

```diff
@@ -247,9 +247,11 @@ function BundleTransaction(transaction, index: number) {
                 }
                 <b>{ coin }</b>
               </a>
-              { coins[coin].value > 0 ? " " + coins[coin].value : "" }
-              { coins[coin].ethValue > 0 ? " ($"+coins[coin].ethValue +")" : "" }
-              { coins[coin].value ? "" : ` (${coins[coin].event})` }
+              <span style={{ marginLeft: coins[coin].ethValue > 0 ? 17 : 0}}>
```

Comment:
I think this is not actually needed (and to be honest, it breaks the alignment between coins)
![image](https://user-images.githubusercontent.com/1703831/121567215-b45b0800-c9f4-11eb-98db-78cca6cffef3.png)

I think it suffices with adding the `min-w-max` class to the parent `<a>`:

```JSX
<a key={ "a_" + index + now() } className="flex hover:underline min-w-max" target="_blank" rel="noreferrer" href={`https://etherscan.io/address/${ coins[coin].address }`} style={{ margin: 3}}>
```

Reply:
just to avoid seeing react warnings:
--------------------------------------
Warning: Each child in a list should have a unique "key" prop.

Reply:
I don't understand, this line does have a prop (and had it from before, I'm just suggesting a CSS class to be added)

Reply:
got it

Reply:
got it, yup can remove the key